core/src/com/unciv/ui/screens/worldscreen/unit/actions/UnitActionsFromUniques.kt

```diff
@@ -165,6 +165,18 @@ object UnitActionsFromUniques {
         ))
     }
 
+    // Instead of Withdrawing, stand your ground!
+    internal fun getGuardActions(unit: MapUnit, tile: Tile): Sequence<UnitAction> {
+        if (!unit.hasUnique(UniqueType.WithdrawsBeforeMeleeCombat)) return emptySequence()
+        return sequenceOf(UnitAction(UnitActionType.Guard,
+            useFrequency = 0f,
+            action = {
+                unit.action = UnitActionType.Guard.value
+                unit.useMovementPoints(unit.currentMovement)
+            }.takeIf { unit.hasMovement() })
+        )
+    }
```

Comment:
I'm not sure about this, since this means a unit can either fortify (to get the defensive bonus) OR guard (to not retreat) but not both
I'm not sure what a good solution would be tbh 🤔

Reply:
A possible solution is to always apply the fortify bonus to units that didn't move. Afterall, how many situations are you standing in place and don't want the fortify bonus?